proposed/2020/Transitive-Dependencies.md

```diff
@@ -75,6 +75,37 @@ When a user highlights a transitive package, they will see a pop-up that display
 
 Finally, selecting a transitive dependency will provide the user the ability to “Install” the package as you normally would a top-level dependency. This will promote the transitive dependency to a top-level dependency, and the transitive dependency will now be placed in the top-level packages list instead.
 
+#### Detailed design
+
+**Collapsible vertical tabs:** Direct and transitive packages will be grouped under collapsible tabs 
```

Comment:
```suggestion
**Collapsible, vertical list group:** Direct and transitive packages will be grouped under collapsible list group
```

Reply:
To not confuse the list groups with the tabs at the top of the PMUI, what if we rename all references to these tabs to "list group" instead or alternatively "list category"?